db/seeds.rb

```diff
@@ -1,8 +1,162 @@
+#!/bin/env ruby
+# encoding: utf-8
+
 # frozen_string_literal: true
+
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
 #
 # Examples:
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+Page.new
```

Comment:
Is this line necessary?

Reply:
genuinely unsure, it was't working without it before but idk if thats still true now